### PR TITLE
TrackSync using websocket and docker network

### PR DESCRIPTION
**Pull request by stdevMac**
## Changes:
- Added connection using websockets for trackSync
- Added params for trackSync
- Solve trackSync issue on macOS

## Types of changes
- [x] Bugfix (non-breaking change which fixes an issue)

## Testing
**Requires testing**

- [x] Yes
- [ ] No

**In case you checked yes, did you write tests?**

- [x] Yes
- [ ] No

### CODE REVIEW

File: templates/services/merge/consensus/teku.tmpl [hunk 1]
```diff
@@ -12,6 +12,8 @@
         condition: service_completed_successfully{{end}}{{end}}
     networks:
       - sedge
+    depends_on:
+      - monitor
```
Comment (adriantpaez) [inline templates/services/merge/consensus/teku.tmpl:16]: Do the other CLs need this the same?

File: cli/cli.go [hunk 2]
```diff
@@ -64,6 +61,7 @@ type CliCmdFlags struct {
 	clExtraFlags      *[]string
 	vlExtraFlags      *[]string
 	logging           string
+	trackSyncWait     int
```
Comment (adriantpaez) [inline cli/cli.go:64]: Please use `int64` instead of `int` because it's the base type of `time.Duration` and all the operations could be done without cast

File: cli/cli_utils.go [hunk 3]
```diff
@@ -16,39 +16,41 @@ limitations under the License.
...
+	Track(done chan bool, consensusUrl, executionUrl []string, wait time.Duration, response chan responseStruct) error
 }
 
-var monitor MonitoringTool
+type containerIPFetcher func(service string, flags *CliCmdFlags) (ip string, err error)
 
-func initMonitor(builder func() MonitoringTool) {
-	monitor = builder()
-}
+var ContainerIPFetcher containerIPFetcher = getContainerIP
```
Comment (adriantpaez) [inline cli/cli_utils.go:51]: Why is this necessary? Can't just use `getContainerIP` directly?

File: cli/cli.go (hunk 2 above)
Comment (stdevMac) [reply cli/cli.go]: done

File: templates/services/merge/consensus/teku.tmpl (hunk 1 above)
Comment (stdevMac) [reply templates/services/merge/consensus/teku.tmpl:16]: I only added it on the execution clients
